gamestonk_terminal/economy/nasdaq_model.py

```diff
@@ -33,12 +33,12 @@ def check_country_code_type(list_of_codes: str) -> List[str]:
 
 
 @log_start_end(log=logger)
-def get_big_mac_index(country_code: str) -> pd.DataFrame:
+def get_big_mac_index(count_code: str) -> pd.DataFrame:
```

Comment:
why this change?

Reply:
It was over the line limit so flake8 was throwing an error.

Reply:
Lol. Break it up onto multiple lines.